app/views/layouts/_footer.html.erb

```diff
@@ -52,7 +52,7 @@
           <div class="col-6">
             <small class="text-left footer-sponsor-logo-text">Hosting Sponsor:</small>
             <a href="http://digitalocean.com/" target="_blank">
```

Comment:
@pranay-simejia Seems like we missed to update the link

Reply:
Apologies to bother you with this issue. Fixed it in #2249 .